pkg/api/networkservice/mechanisms/raw/constants.go

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2020 Doc.ai and/or its affiliates.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package raw provides a Mechanism for raw, no-mechanism connection
+package raw
```

Comment:
Could you please explain why do we need a raw mechanism?

Reply:
For some VFIO cases there can be a network device listening on the endpoint side. So we don't need any interface to be configured on request on the endpoint side - it is already existing.